app_vue/src/layout/navigationBar.vue

```diff
@@ -120,6 +138,27 @@
     }
   }
 
+  :deep .wav-logo {
+    position: relative;
+    width: 220px;
+    height: 50.18px;
+    left: 13%;
+    margin-bottom: 5px;
+  }
+
+  :deep .bottom-logo {
+    left: 48%;
+    width: 310px;
+    height: 81.72px;
+  }
+
+  :deep #bottom-section {
+    position: absolute;
+    bottom: 5%; 
+    width: 82%;
+    left: 10%;
+  }
```

Comment:
lets avoid using `id` unless really necessary! i believe we can use a class instead here

Reply:
done with my change!